src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -174,16 +174,28 @@ public void NotifyDebug()
             // file. However, we leave this here for assurances.
             LastDebugNotify = DateTime.UtcNow;
 
-            // create or update the debug sentinel file to trigger a
-            // debug timeout update across all instances
-            string debugSentinelFileName = Path.Combine(ScriptConfig.RootLogPath, "Host", ScriptConstants.DebugSentinelFileName);
-            if (!File.Exists(debugSentinelFileName))
+            try
             {
-                File.WriteAllText(debugSentinelFileName, "This is a system managed marker file used to control runtime debug mode behavior.");
+                // create or update the debug sentinel file to trigger a
+                // debug timeout update across all instances
+                string debugSentinelFileName = Path.Combine(ScriptConfig.RootLogPath, "Host", ScriptConstants.DebugSentinelFileName);
+                if (!File.Exists(debugSentinelFileName))
+                {
+                    File.WriteAllText(debugSentinelFileName, "This is a system managed marker file used to control runtime debug mode behavior.");
+                }
+                else
+                {
+                    File.SetLastWriteTimeUtc(debugSentinelFileName, DateTime.UtcNow);
+                }
             }
-            else
+            catch (Exception ex)
             {
-                File.SetLastWriteTimeUtc(debugSentinelFileName, DateTime.UtcNow);
+                // best effort
```

Comment:
Making this best effort. Since a call to this api happens on all our admin api routes, errors were causing admin api requests to fail.